doc/multiversion.md

```diff
@@ -0,0 +1,107 @@
+# Multiversion
+
+The default versioning approach for Tarantool Cartridge applications is quite simple:
+
+* only one version of application is installed on machine;
+* updating instance happens on instance restart.
+
+Such approach doesn't protect from accidental updating instance, that can lead to
+errors (for example, if router starts to use new schema earlier than storages).
+The solution is using multiversion approach (currently ot's fully supported only
+for [TGZ](./tgz.md) packages).
+
+Multiversion approach:
+
+* several versions of application can be installed on machine;
+* each instance uses fixed version - it is achieved by using symbolic links;
+* updating instance is (see [`update_instance` step](./scenario.md#update_instance)):
+  * moving it's link to a newest version of application;
+  * instance restart.
+
+## Configuration
+
+All versions of application are placed in `<cartridge_install_dir>/<app-name>-<version>`
+directory.
+
+Symbolic links for instances are placed in `cartridge_instances_dir/<app-name>.<instance-name>`
```

Comment:
```suggestion
Symbolic links for instances are placed in `<cartridge_instances_dir>/<app-name>.<instance-name>`
```

Reply:
Changed to `{{ cartridge_instances_dir }}`.

Reply:
Great!